src/pages/DetailsPage/DetailsPage.js

```diff
@@ -3,43 +3,41 @@ import FavorizeButton from '../../components/FavorizeButton/FavorizeButton'
 import BackButton from '../../components/BackButton/BackButton'
 import { PageContainer } from '../../components/PageContainer/PageContainer'
 import { useParams } from 'react-router'
-import { useRecoilState, useResetRecoilState } from 'recoil'
+import { useRecoilState } from 'recoil'
 import { detailedPodcastState } from '../../states'
 import podcasts from '../../test_data_space.json'
-import { useEffect } from 'react'
+import { useEffect, useState } from 'react'
+import LoadingScreen from '../../components/Loading Screen/LoadingScreen'
 
 export default function DetailsPage() {
   const { id } = useParams()
   const [podcastDetails, setPodcastDetails] = useRecoilState(
     detailedPodcastState
   )
-  const reset = useResetRecoilState(detailedPodcastState)
+  const [isLoading, setIsLoading] = useState(true)
 
   useEffect(() => {
     setPodcastDetails(podcasts.find(podcast => podcast.id === Number(id)))
-
-    return () => {
-      reset()
-    }
+    setTimeout(() => setIsLoading(false), 400)
```

Comment:
It's good to simulate loading, but you might have to remove it in the future when you implement a real API (because you won't know how long it will load).

Reply:
Yes that is right. I will remove that line when i implement the fetch. At the moment it is there to stop the fast flickering screen when clicking on this page.